internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/public.go

```diff
@@ -295,9 +295,14 @@ func (h *Handler) isPublicShare(r *http.Request, oid string) (*link.PublicShare,
 			},
 		},
 	})
-	if err != nil {
-		logger.Err(err)
+	switch {
+	case err != nil:
+		log.Err(err).Send()
 		return nil, false
+	case psRes.Status.Code == rpc.Code_CODE_OK:
+		fallthrough
+	case psRes.Status.Code == rpc.Code_CODE_NOT_FOUND:
+		log.Error().Str("message", psRes.GetStatus().GetMessage()).Str("code", psRes.GetStatus().GetCode().String()).Msg("isPublicShare received unexpected status")
```

Comment:
Thasa two might be too restrictive but woe could change this to only log on internal errors ...

Reply:
Good idea. Internal Errors should be logged on error level. We can log the others on info or debug level but I would like not to spam the error log with unnecessary messages.

So: Yes, please only log internal errors on error level